theme/templates/accounts/account_login.html

```diff
@@ -31,6 +31,9 @@
                 {% include "includes/checkbox_field.html" with field=form.password  field_classes="col-md-12" %}
             </div>
 
+            <!-- robline: putting this here to test oauth login -->
+            <input type="hidden" name="next" value="{{ request.GET.next }}" />
```

Comment:
should this hidden input line for testing be removed before merging since it is only for testing but not used in code?